Exercises/FizzBuzz.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+
+def isMultipleThree(x):
+    if x%3==0:
+        return 'Fizz'
+    else:
+        return x
+
+def isMultipleFive(x):
+    if x%5==0:
+        return 'Buzz'
+    else:
+        return x
+
+def isMultipleSeven(x):
+    if x%7==0:
+        return 'Bang'
+    else:
+        return x
+
+def isMultipleEleven(x):
+    if x%11==0:
+        return 'Bong'
+    else:
+        return x
+        
+def isMultipleThirteen(x):
+    if x%13==0:
+        return 'Fezz'
+    else:
+        return x
+
+
+x=1
+# Take a argument in commandline 
+parser = argparse.ArgumentParser(description='Enter some integer.')
```

Comment:
"Enter some integer" isn't that descriptive. It is specifically the maximum number your fizz buzz script will run on.

Similarly the variable called `inp` should probably be called something like `maximum`